core/probePlugin/src/main/scala/org/virtuslab/ideprobe/Screenshot.scala

```diff
@@ -9,9 +9,9 @@ object Screenshot {
   private val id = new AtomicInteger(0)
 
   private val probeDisplay = sys.env.get("DISPLAY")
-  private val testSuiteName = getEnvOrDefault("IDEPROBE_TEST_SUITE")
-  private val testCaseName = getEnvOrDefault("IDEPROBE_TEST_CASE")
-  private val screenshotsDir = getEnvOrDefault("IDEPROBE_SCREENSHOTS_DIR")
+  private val testSuiteName = sys.env.getOrElse("IDEPROBE_TEST_SUITE", "")
+  private val testCaseName = sys.env.getOrElse("IDEPROBE_TEST_CASE", "")
+  private val screenshotsDir = sys.env.getOrElse("IDEPROBE_SCREENSHOTS_DIR", "")
```

Comment:
Hmmm so if `IDEPROBE_SCREENSHOTS_DIR` is missing... then I suppose `Paths.get("")` resolves to current directory?

Reply:
yes

Reply:
Okay... so in case none of the 3 variables are set, all screenshots will basically be saved to the current directory? 🤔 if so... that'll be rarely a desired outcome of the user, I'd rather use sth like the old `/tmp/ide-probe/screenshots` for that case